File: example-app/src/com/bugsnag/android/example/ExampleActivity.java
package com.bugsnag.android.example;

import android.os.Bundle;

import com.bugsnag.android.BugsnagActivity;

/** If you make your activity inherit from BugsnagActivity, like this one,
    you will get see the correct context of the exception in Bugsnag automatically. **/
public class ExampleActivity extends BugsnagActivity
{
	@Override
	public void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);
		setContentView(R.layout.main);

		// Cause a RuntimeException
		throw new RuntimeException("It broke");
	}
}

File: examples/sdk-app-example/app/src/main/java/com/example/bugsnag/android/ExampleActivity.kt
package com.example.bugsnag.android

import android.os.Bundle
import android.view.View

class ExampleActivity : BaseCrashyActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        findViewById<View>(R.id.multi_process_header).visibility = View.VISIBLE
        findViewById<View>(R.id.multi_process_start_btn).visibility = View.VISIBLE
    }
}
